### PR TITLE
Local environment through single dockerfile

### PR DESCRIPTION
**Pull request by retan38**
The changes introduce a local setup on a single dockerfile that starts a postgresql and redis server besides the web application. This should allow for a simple setup by only running a docker build and docker run commands.

### CODE REVIEW

File: ruqqus/__main__.py [hunk 1]
```diff
@@ -33,6 +33,9 @@
 
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['SQLALCHEMY_DATABASE_URI'] = environ.get("DATABASE_URL")
+
+print(app.config['SQLALCHEMY_DATABASE_URI'])
```
Comment (shamilovtim) [inline ruqqus/__main__.py]: Think you left this in

Comment (retan38) [reply ruqqus/__main__.py]: That should be removed in one of my two additional commits removing debug statements :)

**Comment by retan38** [comment]
I added another commit, now there is a fully working environment with the docker build, feel free to try it!

**Comment by Toxu-ru** [comment]
@retan38  Works great!

An hour ago, I installed it both locally in the usual way and through Docker.  The user would have to give more administrator rights for more complete debugging. (but it's probably not here).

**Comment by captainmeta4** [comment]
Redundant

**Pull request closed without merging**